<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_Main</class>
 <widget class="QMainWindow" name="Window_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>593</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>fw-fanctrl gui</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/icon-256.ico</normaloff>:/icons/icon-256.ico</iconset>
  </property>
  <widget class="QWidget" name="Layout_Main">
   <widget class="QLabel" name="Label_Strategy">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Current strategy:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_ReloadStrategy">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>34</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Font Awesome 6 Free</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_ChangeStrategy">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>85</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="TextArea_Error">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>800</width>
      <height>45</height>
     </rect>
    </property>
    <property name="plainText">
     <string>ERROR_MESSAGE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ComboBox_Strategy">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>85</width>
      <height>34</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>SELECTED_STRATEGY</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Button_Reset">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>34</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Font Awesome 6 Free</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Reset</string>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Resume">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>34</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Font Awesome 6 Free</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Play</string>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Pause">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>34</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Font Awesome 6 Free</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Pause</string>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
